src/ServiceContainer/SymfonyExtension.php

```diff
@@ -151,6 +168,37 @@ public function process(ContainerBuilder $container): void
     {
     }
 
+    /**
+     * @param ContainerBuilder $container
+     * @param array $userConfig
+     * @return array
+     */
+    private function autoconfigure(ContainerBuilder $container, array $userConfig) {
+
+        $defaults = self::SYMFONY_DEFAULTS;
+
+        $symfony4KernelPath = sprintf('%s/%s', $container->getParameter('paths.base'), self::SYMFONY_4_DEFAULTS['kernel']['path']);
```

Comment:
you should avoid this kind of abbreviation 4 => for

Reply:
This is not true, you can put the symfony kernel where you want. The only thing that allow to know the version of symfony is a new parameter of the bootstrap parameter because it does not exist in SF4

Reply:
missing return type hint :array

Reply:
@arnolanglade I have added Symfony 4 configuration defaults when `bootstrap: ~` is set and provided additional information in README as this is indeed an improvement and better DX.

Still, I want to also keep kernel file-based detection to provide zero-configuration setup for those using new default Symfony 4 structure.